core/org.openjdk.jmc.flightrecorder.rules.jdk/src/main/resources/org/openjdk/jmc/flightrecorder/rules/jdk/messages/internal/messages.properties

```diff
@@ -84,9 +84,9 @@ BiasedLockingRevocationRule_TEXT_EPILOGUE=<p>Biased locking is a technique used
 BiasedLockingRevocationPauseRule_CONFIG_WARNING_LIMIT=Biased locking warning limit time
 BiasedLockingRevocationPauseRule_CONFIG_WARNING_LIMIT_LONG=The total time spent on revoking biased locks needed to trigger a warning
 BiasedLockingRevocationPauseRule_RULE_NAME=Biased Locking Revocation Pauses
-BiasedLockingRevocationPauseRule_TEXT_INFO_LONG=To avoid this, either turn off biased Locking ('-XX:-UseBiasedLocking') or, if this occurs during startup, delay the use of biased Locking ('-XX:BiasedLockingStartupDelay').
+BiasedLockingRevocationPauseRule_TEXT_INFO_LONG=Biased locks are an optimization for uncontended locks. When locks have contentions, optimization need to be revoked when application halts. To avoid this, either turn off biased Locking ('-XX:-UseBiasedLocking') or, if this occurs during startup, delay the use of biased Locking ('-XX:BiasedLockingStartupDelay').
```

Comment:
Alternative wording:

Biased locking is an optimization for uncontended locks. If the locks become contended, this optimization must be revoked, which in turn cause application halts. To avoid this, either turn off biased locking ('-XX:-UseBiasedLocking') or, if this occurs during startup, delay the use of biased locking ('-XX:BiasedLockingStartupDelay').